Library/ViewModels/ManagePledgeViewModel.swift

```diff
@@ -167,14 +180,25 @@ public final class ManagePledgeViewModel:
       .map { $0.cancelable }
       .filter(isFalse)
 
-    self.showErrorBannerWithMessage = cancelBackingDisallowed
+    let attemptedDisallowedCancelBackingMessage = cancelBackingDisallowed
       .takeWhen(cancelPledgeSelected)
       .map { _ in
         // swiftformat:disable wrap
         Strings.We_dont_allow_cancelations_that_will_cause_a_project_to_fall_short_of_its_goal_within_the_last_24_hours()
         // swiftformat:enable wrap
       }
 
+    let networkErrorMessage = Signal.merge(
+      fetchProjectEvent.errors().ignoreValues(),
+      graphBackingEvent.errors().ignoreValues()
+    )
+    .map { _ in Strings.Something_went_wrong_please_try_again() }
```

Comment:
Do we have better messaging? We may also need to show something on the view when this happens. Currently it remains blank.